### PR TITLE
ez - added reviewFixtures

### LINKED ISSUE #27: Frontend - Create Review Fixtures
Create reviewFixtures #2

### PR DESCRIPTION
**Pull request by emiz39**
Closes #27 

Part of Frontend #2

### CODE REVIEW

**Review by clevermonkey16 — DISMISSED** [review]

File: frontend/src/fixtures/reviewFixtures.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const reviewFixtures = {
+  oneReview: {
+    id: 1,
+    item: "Buttermilk Fried Chicken Sandwich",
+    station: "Entree Specials",
+    stars: "***",
+    reviewText: "soggy",
+    status: "Awaiting Moderation",
+    modComments: "",
```
[the remark this answers is not in the document]
Comment (emiz39) [reply frontend/src/fixtures/reviewFixtures.js:11]: fixed

File: frontend/src/fixtures/reviewFixtures.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const reviewFixtures = {
+  oneReview: {
+    id: 1,
+    item: "Buttermilk Fried Chicken Sandwich",
+    station: "Entree Specials",
+    stars: "***",
```
[the remark this answers is not in the document]
Comment (emiz39) [reply frontend/src/fixtures/reviewFixtures.js]: fixed

File: frontend/src/fixtures/reviewFixtures.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const reviewFixtures = {
+  oneReview: {
+    id: 1,
+    item: "Buttermilk Fried Chicken Sandwich",
```
[the remark this answers is not in the document]
Comment (emiz39) [reply frontend/src/fixtures/reviewFixtures.js]: fixed

File: frontend/src/fixtures/reviewFixtures.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  oneReview: {
+    id: 1,
+    reviewerId: 101,
+    itemId: 201,
+    dateServed: "2023-11-01T12:00:00",
+    stars: 3,
+    reviewText: "soggy",
+    status: "Awaiting Moderation",
+    modId: null,
+    modComments: "",
```
Comment (st4lemon) [inline frontend/src/fixtures/reviewFixtures.js:11]: I would make modComments null if awaiting approval, so we can differentiate between unreviewed vs no comments

**Review by st4lemon — CHANGES_REQUESTED** [review]
other than modComments=null, LGTM

File: frontend/src/fixtures/reviewFixtures.js (hunk 4 above)
Comment (emiz39) [reply frontend/src/fixtures/reviewFixtures.js]: fixed

File: frontend/src/fixtures/reviewFixtures.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  oneReview: {
+    id: 1,
+    reviewerId: 101,
+    itemId: 201,
+    dateServed: "2023-11-01T12:00:00",
+    stars: 3,
+    reviewText: "soggy",
+    status: "Awaiting Moderation",
+    modId: null,
+    modComments: "null",
```
Comment (st4lemon) [inline frontend/src/fixtures/reviewFixtures.js:11]: null not "null" :(

**Review by st4lemon — CHANGES_REQUESTED** [review]

File: frontend/src/fixtures/reviewFixtures.js (hunk 5 above)
Comment (emiz39) [reply frontend/src/fixtures/reviewFixtures.js]: fixed :D

File: frontend/src/fixtures/reviewFixtures.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  oneReview: {
+    id: 1,
+    reviewerId: 101,
+    itemId: 201,
+    dateServed: "2023-11-01T12:00:00",
+    stars: 3,
+    reviewText: "soggy",
+    status: "Awaiting Moderation",
+    modId: null,
+    modComments: null,
```
Comment (st4lemon) [inline frontend/src/fixtures/reviewFixtures.js:11]: wonderful

**Review by st4lemon — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**